yarr@jtoberling/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - 2023-10-13
+- Initial version
+
+## [1.0.1] - 2024-01-16
+- Fix byteArray import bug [#1036](https://github.com/linuxmint/cinnamon-spices-desklets/issues/1036)
```

Comment:
Just a heads up that you typically want the latest version at the top of the CHANGELOG file.